clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-m5d4x_machine_set.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: build01-9hdwj
+  name: build01-9hdwj-worker-us-east-1a-m5d4x
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
```

Comment:
I specified `replicas: 1` since those m5d4x machinesets are not linked to machineautoscaler.